src/hct_mis_api/apps/household/models.py

```diff
@@ -938,85 +937,152 @@ class Individual(
+        max_length=50, choices=SEVERITY_OF_DISABILITY_CHOICES, blank=True, help_text="Selfcare disability"
+    )
+    comms_disability = models.CharField(
+        max_length=50, choices=SEVERITY_OF_DISABILITY_CHOICES, blank=True, help_text="Comms disability"
+    )
+
+    who_answers_phone = models.CharField(max_length=150, blank=True, help_text="Who answers phone number")
+    who_answers_alt_phone = models.CharField(max_length=150, blank=True, help_text="Who answers alt phone number")
+
+    preferred_language = models.CharField(
+        max_length=6, choices=Languages.get_tuple(), null=True, blank=True, help_text="Preferred language"
     )
-    first_registration_date = models.DateField()
-    last_registration_date = models.DateField()
-    flex_fields = JSONField(default=dict, blank=True, encoder=FlexFieldsEncoder)
-    enrolled_in_nutrition_programme = models.BooleanField(null=True)
-    administration_of_rutf = models.BooleanField(null=True)
+    relationship_confirmed = models.BooleanField(default=False, help_text="Relationship confirmed status")
+    wallet_name = models.CharField(max_length=64, blank=True, default="", help_text="Wallet name")
+    blockchain_name = models.CharField(max_length=64, blank=True, default="", help_text="Blockchain name")
+    wallet_address = models.CharField(max_length=128, blank=True, default="", help_text="Wallet address")
+
+    # System fields
+    individual_collection = models.ForeignKey(
+        IndividualCollection,
+        related_name="individuals",
+        on_delete=models.CASCADE,
+        null=True,
+        help_text="Collection of individual representations [sys]",
+    )
+    duplicate = models.BooleanField(default=False, db_index=True, help_text="Duplicate status [sys]")
+    duplicate_date = models.DateTimeField(null=True, blank=True, help_text="Duplicate date [sys]")
+    withdrawn = models.BooleanField(default=False, db_index=True, help_text="Withdrawn status [sys]")
+    withdrawn_date = models.DateTimeField(null=True, blank=True, help_text="Withdrawn date [sys]")
+
+    phone_no_valid = models.BooleanField(null=True, db_index=True, help_text="Beneficiary phone number valid [sys]")
+    phone_no_alternative_valid = models.BooleanField(
+        null=True, db_index=True, help_text="Beneficiary phone number alternative valid [sys]"
+    )
+    registration_data_import = models.ForeignKey(
+        "registration_data.RegistrationDataImport",
+        related_name="individuals",
+        on_delete=models.CASCADE,
+        help_text="RDI where Beneficiary was imported [sys]",
+    )
+    first_registration_date = models.DateField(help_text="First registration date [sys]")
+    last_registration_date = models.DateField(help_text="Last registration date [sys]")
+    enrolled_in_nutrition_programme = models.BooleanField(null=True, help_text="Enrolled in nutrition programe [sys]")
+    administration_of_rutf = models.BooleanField(null=True, help_text="Administration on rutf [sys]")
```

Comment:
?

Reply:
I don't know what this field means and where we used it honestly in app
![image](https://github.com/user-attachments/assets/43ddc391-5141-41de-b91b-1cfd065d4582)

Reply:
@johniak do you know if we still need this field `administration_of_rutf ` ?

Reply:
@domdinicola should we remove it as it comes from CashPlan?
https://github.com/unicef/hope/commit/3155e4f6dc372454995d317ed3c7af6b0aae16c6